tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -35,8 +35,12 @@ class InactiveUser(Exception):
 class TokenAuthorizationOIDC(ObtainAuthToken):
     """Define methods for handling login request from login.gov."""
 
-    def decode_payload(self, id_token):
+    @staticmethod
+    def decode_payload(id_token, options=None):
```

Comment:
The addition of the optional argument `options` was added to support disabling signature verification in tests. The default should be used in all cases other than testing.